tools/sfxgen/sfxgen.py

```
@@ -0,0 +1,178 @@
+    if args.command == "generate":
+        api_key = os.environ.get("ELEVENLABS_API_KEY")
+        if not api_key:
+            print("error: ELEVENLABS_API_KEY is not set", file=sys.stderr)
+            return 2
+        failures = []
+        for sound in load_sounds(args.manifest):
+            if args.only and sound["file"] != args.only:
+                continue
+            try:
+                generate_one(sound, api_key, out_dir)
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)